### ISSUE TITLE
Single finger gestures not recognized

### ISSUE THREAD

**Issue by ohthehugemanatee** [xiamaz/libinput-touchscreen#2]
Thank you for your work on this! It's been a big help to me in my own workflows.

I've noticed that single finger gestures are not picked up as gesture . In config:

```
GESTURE W 1
    ydotool key Ctrl+PageUp
```

Never registers as a gesture, though `GESTURE W 2` and more fingers work fine... and so does `BORDER W 1`. If thsi is intentional because of scrolling, it should be documented.

**Comment by xiamaz** [comment]
Yes, this might have been somewhat intentional when I implemented it, but I will look into it.

Sorry for the lack of documentation, it was only a very hacky tool that I cobbled together because of the lack of tools to work with libinput gestures that are exposed perfectly fine.
